packages/service-clients/tinylicious-client/.eslintrc.cjs

```diff
@@ -8,4 +8,14 @@ module.exports = {
 	parserOptions: {
 		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
 	},
+	overrides: [
+		{
+			// Overrides for tests
+			files: ["src/test/*.spec.ts"],
+			rules: {
+				// https://mochajs.org/#arrow-functions
```

Comment:
Can we add a comment explaining the rationale for disabling this for tests?

Reply:
The URL in the comment (https://mochajs.org/#arrow-functions) describes the rationale - not sure if you saw that or if you're looking for more text directly in this file?

Reply:
Oh, I assumed the link was just for more general context on the rule; I didn't look. I think we should add an explicit note here as a comment (fine if we want to say "see this link for rationale for disabling in tests" or something). I'd just like for any rule deviations to have clear rationale in the docs so devs know if/when they can be changed.

Reply:
Added a note in latest